engine/engine/src/engine.cpp

```diff
@@ -935,6 +935,11 @@ namespace dmEngine
         gui_params.m_GetUserDataCallback = dmGameSystem::GuiGetUserDataCallback;
         gui_params.m_ResolvePathCallback = dmGameSystem::GuiResolvePathCallback;
         gui_params.m_GetTextMetricsCallback = dmGameSystem::GuiGetTextMetricsCallback;
+
+        // If window size changed at the initialization phase, size should be actualized
```

Comment:
What "phase" specifically?
I believe that should be put in the code comment

Reply:
true!
```
// If an extension changes window size at extensions initialization phase, engine should read that.
```